torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1065,22 +1077,22 @@ def elastic_bounding_box(
     # Or add spatial_size arg and check displacement shape
     spatial_size = displacement.shape[-3], displacement.shape[-2]
 
-    id_grid = _FT._create_identity_grid(list(spatial_size)).to(bounding_box.device)
+    id_grid = _create_identity_grid(spatial_size, bounding_box.device)
     # We construct an approximation of inverse grid as inv_grid = id_grid - displacement
     # This is not an exact inverse of the grid
-    inv_grid = id_grid - displacement
+    inv_grid = id_grid.sub_(displacement)
 
     # Get points from bboxes
     points = bounding_box[:, [[0, 1], [2, 1], [2, 3], [0, 3]]].reshape(-1, 2)
-    index_x = torch.floor(points[:, 0] + 0.5).to(dtype=torch.long)
-    index_y = torch.floor(points[:, 1] + 0.5).to(dtype=torch.long)
+    index_xy = points.ceil_().to(dtype=torch.long)
```

Comment:
Better do:
```suggestion
    if points.is_floating_point():
        points = points.ceil_()
    index_xy = points.to(dtype=torch.long)
```

Because without it you can get:
```
vision/torchvision/prototype/transforms/functional/_geometry.py", line 1545, in elastic_bounding_box_b32f7
    index_xy = points.ceil_().to(dtype=torch.long)
RuntimeError: "ceil_vml_cpu" not implemented for 'Long'
```